build/tasks/bundle.js

```diff
@@ -101,7 +101,8 @@ gulp.task("bundle:clarity:js", ["typescript:clarity"], function () {
             "rxjs*": {build: false}
         },
         packages: {
-            'clarity-angular': {main: 'index.js', defaultExtension: 'js'}
+            'clarity-angular': {main: 'index.js', defaultExtension: 'js'},
+            'clarity-icons': {defaultExtension: 'js'}
```

Comment:
reverted as per my other comment about putting all icons that clarity components depend on into core (vs essential).